Sources/Plasma/FeatureLib/pfDXPipeline/plDXPipeline.cpp

```diff
@@ -4914,26 +4914,26 @@ void plDXPipeline::IRestoreSpanLights()
 void    plDXPipeline::IScaleD3DLight( plDXLightRef *ref, float scale )
 {
     scale = int(scale * 1.e1f) * 1.e-1f;
-    if( ref->fScale != scale )
-    {
-        D3DLIGHT9       light = ref->fD3DInfo;
+    if (ref->fScale == scale)
+        return;
 
+    D3DLIGHT9       light = ref->fD3DInfo;
 
-        light.Diffuse.r *= scale;
-        light.Diffuse.g *= scale;
-        light.Diffuse.b *= scale;
 
-        light.Ambient.r *= scale;
-        light.Ambient.g *= scale;
-        light.Ambient.b *= scale;
+    light.Diffuse.r *= scale;
+    light.Diffuse.g *= scale;
+    light.Diffuse.b *= scale;
 
-        light.Specular.r *= scale;
-        light.Specular.g *= scale;
-        light.Specular.b *= scale;
+    light.Ambient.r *= scale;
+    light.Ambient.g *= scale;
+    light.Ambient.b *= scale;
 
-        fD3DDevice->SetLight( ref->fD3DIndex, &light );
-        ref->fScale = scale;
-    }
+    light.Specular.r *= scale;
+    light.Specular.g *= scale;
+    light.Specular.b *= scale;
+
+    fD3DDevice->SetLight( ref->fD3DIndex, &light );
+    ref->fScale = scale;
```

Comment:
There's a few of these in the source code. I don't know if H'uru has a stance on this. Early return vs single exit is its own debate in industry. Because other code sections also use early return - my hunch is it won't be an issue.